packages/webapp/src/containers/Task/TaskDate/index.jsx

```diff
@@ -10,11 +10,20 @@ function TaskDate({ history, match, location }) {
     history.back();
   };
   const isTransplantTask = useIsTaskType('TRANSPLANT_TASK');
+  const isMovementTask = useIsTaskType('MOVEMENT_TASK');
 
   const tasks = location.state.management_plan_id
     ? useSelector(tasksByManagementPlanIdSelector(location.state.management_plan_id))
     : [];
 
+  const getNextStepPath = () => {
+    return isTransplantTask
+      ? '/add_task/task_crops'
+      : isMovementTask
+        ? '/add_task/task_animal_selection'
+        : '/add_task/task_locations';
```

Comment:
I think a switch would be more scalable here if we need to add logic for other tasks

```js
const taskType = useSelector(taskTypeSelector(persistedFormData.task_type_id));

const getNextStepPath = () => {
    switch (taskType) {
        case 'TRANSPLANT_TASK':
            return ...
        case 'MOVEMENT_TASK':
            return ...
        default:
            return ....
    }
}
```

Reply:
@antsgar I wanted to do that... but there was a bit more logic than that in the `useIsTaskType()` function that pre-existed. And that function is used throughout app.  The logic also checks if a custom task is named 'movement_task' too. I thought to just reuse existing for now... but I could refactor the logic for this case.

Reply:
I wouldn't necessarily use that function, it might still be useful for other cases but not this one so I think it's okay to do it differently for this case. That said, this is not a blocker to merge for me!

Reply:
Okay I think I will leave it as is for now if thats ok -- there shouldn't be too many other cases. It seems like movements (transplant and animals) are just a bit different of flow. And I am pretty sure any new function still needs the functionality of `useIsTaskType()` -- which I could extend but not yet sure how valuable that is.